controllers/defaults/placements.go

```diff
@@ -126,6 +131,30 @@ var (
 				getOcsToleration(),
 			},
 		},
+
+		ProviderServerKey: {
```

Comment:
@iamniting We are moving away from running the API server only in provider mode (it has different applications in DR). Can we please call this key something else? maybe `OCSAPIServerKey`

Reply:
i guess "api-server" would make more sense.

Reply:
changed it to the `APIServerKey`